tethysext/atcore/job_scripts/update_resource_status.py

```diff
@@ -0,0 +1,56 @@
+#!/opt/tethys-python
+import sys
+import traceback
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+from tethysext.atcore.models.app_users import Resource
+
+
+def run(resource_db_url: str, resource_id: str, status_keys: list):
+    """
+    Update the root status of a resource based ont he status of one or more other statuses of the same resource.
+
+    Args:
+        resource_db_url (str): The SQLAlchemy to the resource database.
+        resource_id (str): The resource ID.
+        status_keys (list): A list of status keys to check.
```

Comment:
suggestion for status_keys:

status_keys(list): One or more keys of statuses to check to determine resource status. The other jobs must update these statuses to one of the Resource.OK_STATUSES for the resource to be marked as SUCCESS.

Reply:
looks good